src/lightning_app/cli/cmd_clusters.py

```diff
@@ -91,7 +91,16 @@ def create(
         if wait:
             _wait_for_cluster_state(self.api_client, resp.id, V1ClusterState.RUNNING)
 
-        click.echo(f"{resp.id} cluster is in {resp.status.phase} state")
+        click.echo(
+            f"""{resp.id} is now being created... This can take up to an hour.
+
+        To view your clusters status use:
```

Comment:
Is so much indentation intentional?

Reply:
it should not - changed in subsequent commit.